From 9dda18921a25f64e0a0da06e3417ce99f8bf9b41 Mon Sep 17 00:00:00 2001
From: Andrew Clemons <andrew.clemons@gmail.com>
Date: Wed, 25 Oct 2017 21:35:14 +1300
Subject: [PATCH] Silence warning for missing path for `vtcol` binary

vtcol.rs should probably be main.rs, but explicitly setting the
non-standard path also silences the warning. (src/bin would also work)
---
 Cargo.lock | 2 +-
 Cargo.toml | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 16590af..0833d44 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,6 +14,7 @@ license     = "GPL-3.0"
 
 [[bin]]
 name        = "vtcol"
+path        = "src/vtcol.rs"
 test        = false
 doc         = false
 
